tests/nuodb_basic_test.py

```diff
@@ -172,6 +172,23 @@ def test_many_significant_digits(self):
     def test_numeric_no_decimal(self):
         self._test_decimal_fixture(decimal.Decimal("1.000"), 5, 3)
 
+    # This test case produces at least three different defects, perhaps
+    # more under the covers.
+    @unittest.skip("produces data errors for e-notations")
+    def test_enotation_decimal_large(self):
+        numbers = (
+            # Yields:  AssertionError: '400000000.00' != '4E+8'
+            decimal.Decimal('4E+8'),
```

Comment:
And more fancier than both of us know. There's a lot of goodness in there that I have yet to uncover.